### PR TITLE
Activiti/Activiti#3355 adding multi-instance var mapping test

### PR DESCRIPTION
**Pull request by Nannakaroliina**
Ref https://github.com/Activiti/Activiti/issues/3355

### CODE REVIEW

File: activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java [hunk 1]
```diff
@@ -148,6 +150,111 @@ public void processTaskVarMapping() {
...
+                    "date",
+                    datetime)
+            );
+
+        List<VariableInstance> taskVariables = taskBaseRuntime.getTasksVariablesByTaskId(tasks.get(0).getId());
+
+        assertThat(taskVariables)
+            .isNotNull()
+            .extracting(VariableInstance::getName,
+                VariableInstance::getType,
```
Comment (miguelruizdev) [inline activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java:192]: good idea on the check of var type 👍

File: activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java [hunk 2]
```diff
@@ -148,6 +150,111 @@ public void processTaskVarMapping() {
         processBaseRuntime.delete(processInstance.getId());
     }
 
+    @Test
+    public void processTaskVarMappingForMultiInstance() {
```
Comment (miguelruizdev) [inline activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java]: maybe we can update the test names of the whole class to the agreed standard:
`should_somethingHappen_when_somethingElseHappens'
it'll be clearer that way what we are trying to do with each case

**Review by miguelruizdev — CHANGES_REQUESTED** [review]

File: activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java (hunk 2 above)
Comment (miguelruizdev) [reply activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java]: ignore the long method warning: https://app.codacy.com/gh/Activiti/Activiti/pullRequest?prid=5752335

Comment (Nannakaroliina) [reply activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java]: Updated the names for tests

Comment (miguelruizdev) [reply activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java]: Avoid using camel-case after a hyphen/underscore:
`should_NotHaveTaskVariables_when_MappingIsEmpty` 
should be
`should_notHaveTaskVariables_when_mappingIsEmpty`

**Review by miguelruizdev — APPROVED** [review]
LGTM! 👍

**Pull request merged**